labs/lab-08/tasks/sum7/support/sum7.asm

```diff
@@ -0,0 +1,26 @@
+section .text
+
+global sum7
+
+sum7:
+    push rbp
+    mov rbp, rsp
+
+    sub rsp, 8              ; leave room in order to align the stack
+    push rbx                ; preserve rbx, r12-r15 as required by cdecl
+    push r12
+    push r13
+    push r14
+    push r15
```

Comment:
Keep these here because this is the first exercise and it shows students what registers each function is supposed to save.